posts/data-hackathon-recap.md

```diff
@@ -0,0 +1,39 @@
+---
+authors:
+  - Ramya Shetty
+tags:
+  - Data
+date: 2021-02-10T14:50:55.000Z
+title: 'Data Hackathon Recap'
+image: https://raw.githubusercontent.com/ippontech/blog-usa/master/images/2021/02/hackathon_header.png
+---
+Is the Holiday Spirit Contagious? During Ippon's first Data Hackathon in Dec 2020, the Data Practice set out to find the effect of holidays on COVID-19 case count after typically large gatherings on holidays like 4th of July or non-official ones like Black Friday and as a function of geo-location, voter registration, local preventative measures.
+
+We had two brief planning sessions to identify the objective, scope, tech stack, data sources and to organize into smaller groups to tackle the work using, in true hackathon spirit, a python script developed on the spot! A high level architecture included S3, Lambda, SNS, SQS, AWS Glue using Scala, Athena, QuickSight.
+
+![High Level Architecture](https://raw.githubusercontent.com/ippontech/blog-usa/master/images/2021/02/hackathon_high_level_arch.png)
+
+Data sources included
+- COVID-19 Case Counts by state for 2020 [Source](https://data.cdc.gov/Case-Surveillance/United-States-COVID-19-Cases-and-Deaths-by-State-o/9mfq-cb36)
+- Holiday List
+- Voter Data Registration by State
```

Comment:
I think we just ended up using this for population data - maybe add a little something clarifying that?